scope_classification.py

```diff
@@ -0,0 +1,343 @@
+import json
+import urllib
+import requests
+import numpy as np
+import pandas as pd
+from typing import Optional, Mapping
+from penquins import Kowalski
+from time import sleep
+
+# define the baseurl and set the token to connect
+BASE_URL = 'https://fritz.science/'
+
+# put your Fritz token here. You can get it from your Fritz profile page
+token = "<token>"
```

Comment:
Michael, I am a bit confused, since the token is used in the api function but it has been moved to fritz utils file, and the token is passed in the upload_classification file, how can it be used for the api function? or can i just have the api function in both fritz utils and upload_classification?

Reply:
You can pass the token as a command line argument right?

Reply:
Sorry I mean function argument.